web/src/modules/user/wallets/sagas/walletsAddressSaga.ts

```diff
@@ -10,9 +10,11 @@ const walletsAddressOptions: RequestOptions = {
 export function* walletsAddressSaga(action: WalletsAddressFetch) {
     try {
         const currency = action.payload.currency.toLocaleLowerCase();
-        const url = `/account/deposit_address/${currency}`;
-        const { address, currencies, state } = yield call(API.get(walletsAddressOptions), url);
-        yield put(walletsAddressData({ address, currencies, state }));
+        const blockchainKey = action.payload.blockchain_key?.toLocaleLowerCase();
+
+        const url = `/account/deposit_address/${currency}?blockchain_key=${blockchainKey}`;
```

Comment:
what if blockchainKey is undefined?
please make sure you don't send requests with url link `/account/deposit_address/${currency}?blockchain_key=`

Reply:
we have conditions inside the container for it ( blockchain_key it's primary param )